node/manifest.ts

```diff
@@ -44,15 +49,39 @@ interface Route {
 
 const serializePattern = (regex: RegExp) => regex.source.replace(/\\\//g, '/')
 
+const sanitizeEdgeFunctionConfig = (config: Record<string, EdgeFunctionConfig>): Record<string, EdgeFunctionConfig> => {
+  const newConfig = { ...config }
+
+  for (const [name, functionConfig] of Object.entries(newConfig)) {
+    if (functionConfig.excluded_patterns.length === 0) {
+      delete newConfig[name]
```

Comment:
Personally not a big fan of using `delete` here. I would rather start with an empty object for `newConfig` and then add the individual properties if `excluded_patterns.length !== 0`.

Reply:
done in https://github.com/netlify/edge-bundler/pull/274/commits/699a05367ae86b26b3ddf4390a0d103fcad9eba6